### DISCUSSION TITLE
How to delete a site

### DISCUSSION THREAD

**Discussion by babrees** [leokhoa/laragon#558] [Laragon's Articles & How-tos]
As the subject says, how do you delete a site on Laragon?

**Comment by Pen-y-Fan** [comment]
1. open windows explorer
2. navigate to Laragon's www directory
3. delete the project folder
4. open your SQL manager of choice (e.g. HeideSQL) and delete the SQL database (take a backup first)
5. stop and start Laragon.
6. you may need to accept the elevated privileges prompt, for Laragon to recreate the hosts file.
